packages/pwa/src/components/feedSubscriptions/FeedSubscriptionSettings.tsx

```
@@ -218,6 +228,7 @@
 export const FeedSubscriptionSettingsButton: React.FC<{
   readonly userFeedSubscription: UserFeedSubscription;
 }> = ({userFeedSubscription}) => {
+  console.log('userFeedSubscription', userFeedSubscription);
```

ESLint

[error] Unexpected console statement.

(no-console)